src/Tribe/Cache/Cache_Interface.php

```diff
@@ -18,18 +18,22 @@ public function reset_all();
 	 *
 	 * Please note that the list is aware of supported types.
 	 *
+	 * @param array $post_types An array of post types overriding the supported ones.
+	 *
 	 * @return array
 	 */
-	public function posts_without_ticket_types();
+	public function posts_without_ticket_types( array $post_types = null );
```

Comment:
I've changed the only implementation using it for the time being, that's code that I've added a week ago so we are safe in this case.